subprojects/tooling-builders/src/main/kotlin/org/gradle/kotlin/dsl/tooling/builders/KotlinBuildScriptModelBuilder.kt

```diff
@@ -91,9 +95,22 @@ object KotlinBuildScriptModelBuilder : ToolingModelBuilder {
     override fun canBuild(modelName: String): Boolean =
         modelName == "org.gradle.kotlin.dsl.tooling.models.KotlinBuildScriptModel"
 
-    override fun buildAll(modelName: String, modelRequestProject: Project): KotlinBuildScriptModel =
-        scriptModelBuilderFor(modelRequestProject as ProjectInternal, requestParameterOf(modelRequestProject))
-            .buildModel()
+    override fun buildAll(modelName: String, modelRequestProject: Project): KotlinBuildScriptModel {
+        val logger = modelRequestProject.logger
+        val parameter = requestParameterOf(modelRequestProject)
+        try {
+            return kotlinBuildScriptModelFor(modelRequestProject, parameter).also {
+                println("$parameter => $it")
```

Comment:
❓ Why use `println` and not the `logger` instance brought into scope a few lines above?

Reply:
`logger` output from the model builder doesn't end up in the daemon log (tried with `info` and `debug`). The variable is a leftover from a previous iteration. I'll remove it.